daceml/util/utils.py

```diff
@@ -104,6 +105,54 @@ def find_str_not_in_set(existing: typing.Set[str],
     return base_name + "_" + str(i)
 
 
+def vectorize_array_and_memlet(sdfg, array_name, type: dtypes.typeclass):
+    '''
+       Adjust the shape of a data container according to the vec width (only the last dimension).
```

Comment:
Shouldn't -1 (the last dimension) just be a parameter of the function? i.e., "dim: int = -1"?

Reply:
Yeah, but didn't we say that vectorization can occur only on the last dimension?

Reply:
depending on the strides, but ok, we can assume C-contiguousness.